lib/l10n_utils/fluent.py

```diff
@@ -84,6 +84,11 @@ def active_locales(self):
         # first resource is the one to check for activation
         return get_active_locales(self.resource_ids[0])
 
+    @cached_property
+    def active_home_locales(self):
+        # use mozorg/home to check for activation
+        return get_active_locales("mozorg/home.ftl")
```

Comment:
_Originally posted by @janbrasna in https://github.com/mozilla/bedrock/issues/15010#issuecomment-2555966500_:

> This is getting a bit complicated now with three home ftls:
> 
> 1. `mozorg/home-m24.ftl`
> 2. `mozorg/home-new.ftl`
> 3. `mozorg/home.ftl`
> 
> with varying activation metadata.
> 
> Normally a "good enough" check might be for the activation of the oldest `home.ftl` file for home-old (`?xv=legacy`) that is the "lowest common denominator" fallback if nothing better exists, however this is not 100% correct:
> 
> A corner case might be e.g. a locale that never had home-old or home-new translated, but got active recently and completed home-m24 localization — such language should also be listed in the global root as active.

So a correct way might be to set union of all three for good measure?

Or, basically, look at how e.g. the language picker does it for home (paths), to offer all the locales, no matter if they are available for m24, new or old, in one list.